### ISSUE TITLE
Missing Codes

### ISSUE THREAD

**Issue by JX-Gallade** [the404devs/sm64code#3]
Hey, I don't know if you guys are still updating the website,
But I seemed to have run into a bit of a problem.

I used the site about a year or more ago before it was updated, and I wanted to try out some new color codes for the game,
But whether it was Retroarch or Project64, the code never seemed to work correctly, no matter how much I edited it manually. Whenever I tried to input the code, the custom color always merged with the original colors on Mario's Model.

I pulled out a saved cheat file from my folders from when I made preset codes a year ago and noticed a significant difference between the ones I made before and the ones I made now. Some of the codes for the colors were missing.

The new website has four sets of codes for each palette: 2 for the base and 2 for the shadows.
However, looking at the old website version (Using the Wayback Machine), Each palette had eight sets of codes: 4 for the base and 4 for the shadows. 

Example: 

New:

8107EC20 XXXX
8107EC22 XXXX
8107EC28 XXXX
8107EC2A XXXX

Old:

8107EC20 XXXX
8107EC22 XXXX
8107EC24 XXXX
8107EC26 XXXX
8107EC28 XXXX
8107EC2A XXXX
8107EC2C XXXX
8107EC2E XXXX

Editing the codes I received from the old website seemed to have fixed the color code.
I might be too late, But if, by any chance, you guys are still updating the site, I would greatly appreciate it if the site was updated to give the eight sets of codes for each palette instead of the four if I'm making any sense, I hope.

**Comment by the404devs** [comment]
Thanks for bringing this to my attention.

When I rewrote this project a while back, I took out those extra memory addresses as I thought they didn't affect anything. I see now that Mario gets a bit funky if they're not set on PJ64. I never noticed this, as I only tested on Parallel Launcher at the time.

PJ64, 4 codes per Mario part:
![Screenshot_20241205_205301](https://github.com/user-attachments/assets/2aa57701-a69e-42fd-a906-f951fb1e1e1d)

Parallel, 4 codes per Mario part:
![Screenshot_20241205_205307](https://github.com/user-attachments/assets/c5c85e97-3d2a-4267-89eb-c853fa29f4a8)



Fortunately, copying the colour to those extra addresses fixes it in Project64, see below:
![Screenshot_20241205_205846](https://github.com/user-attachments/assets/959f5ce8-f2a7-4330-ad6a-42718f67ea3e)


I'll have this fixed on the site shortly.
